### PR TITLE
crud respondents

### PR DESCRIPTION
**Pull request by krakie**

### CODE REVIEW

File: Library/Models/SampleFilter.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
```
Comment (neilboyd) [inline Library/Models/SampleFilter.cs]: Add the copyright header

File: Library/Services/Implementation/NfieldSurveySampleService.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
+﻿using System;
```
Comment (neilboyd) [inline Library/Services/Implementation/NfieldSurveySampleService.cs]: add the header

File: Library/Models/SampleUploadStatus.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
```
Comment (neilboyd) [inline Library/Models/SampleUploadStatus.cs]: add the file header

File: Library/Models/SampleUploadStatus.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        public int UpdatedCount { get; set; }
+        public int DuplicateKeyCount { get; set; }
+        public int EmptyKeyCount { get; set; }
+        public int InvalidKeyCount { get; set; }
+        public int InvalidDataCount { get; set; }
+        public int SkippedCount { get; set; }
+        public bool HeaderInvalid { get; set; }
+        public int HeaderInvalidColumnsCount { get; set; }
+        public bool HeaderDataMismatch { get; set; }
+    }
```
Comment (neilboyd) [inline Library/Models/SampleUploadStatus.cs]: add comments - you can copy them from the sample package

File: Library/Models/SampleFilter.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+using System;
+
+namespace Nfield.Models
+{
+    internal class SampleFilter
+    {
+        public string Name { get; set; }
+        public string Op { get; set; }
+        public string Value { get; set; }
+    }
```
Comment (neilboyd) [inline Library/Models/SampleFilter.cs]: add comments

File: Examples/NfieldSampleManagement.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            var sample = _surveySampleService.GetAsync(surveyId).Result;
+        }
+
+        /// <summary>
+        /// upload sample
+        /// Note that this example does not return the result, allthough your real class will do so.
+        /// </summary>
+        /// <param name="surveyId">id of the survey where we want to upload the sample records to</param>
+        /// <param name="sample">path of the csv file containing the sample</param>
```
Comment (marcoNipo) [inline Examples/NfieldSampleManagement.cs]: I think this comment is wrong, it says "path of the csv file", but the string is used directly as content when posting

Comment (krakie) [inline Examples/NfieldSampleManagement.cs]: I changed the comment

**Pull request merged**